cms/toolbar/toolbar.py

```diff
@@ -97,7 +98,7 @@ def init_toolbar(self, request):
         self.request = request
         self.is_staff = self.request.user.is_staff
         self.edit_mode = self.is_staff and self.request.session.get('cms_edit', False)
-        self.build_mode = self.is_staff and self.request.session.get('cms_build', False)
+        self.build_mode = self.is_staff and request.GET.get(get_cms_setting('CMS_TOOLBAR_URL__BUILD')) != None
```

Comment:
so here I don't really know what to do (or rather how to do it correctly). if i keep the old code - it works for the first time you visit the the page with `?build=1` but then it _never_ resets back to false.